docs/articles/custom-skins.md

```diff
@@ -28,89 +28,53 @@ Default template
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Default">
             <!-- Background Resources -->
-            <Color x:Key="SolidBackgroundFillColorBase">#FF262738</Color>
-            <Color x:Key="SolidBackgroundFillColorSecondary">#FF3F2E4B</Color>
-            <Color x:Key="SolidBackgroundFillColorTertiary">#FF332041</Color>
-            <Color x:Key="SolidBackgroundFillColorQuarternary">#332041</Color>
+            <SolidColorBrush x:Key="RootBackgroundBrush" Color="#AA7F3C4E" />
+            <Color x:Key="SolidBackgroundFillColorBase">#97475C</Color>
+            <Color x:Key="SolidBackgroundFillColorSecondary">#7F3C4E</Color>
+            <Color x:Key="SolidBackgroundFillColorTertiary">#763B4A</Color>
+            <Color x:Key="SolidBackgroundFillColorQuarternary">#592A37</Color>
             <SolidColorBrush x:Key="SolidBackgroundFillColorBaseBrush" Color="{ThemeResource SolidBackgroundFillColorBase}" />
             <SolidColorBrush x:Key="SolidBackgroundFillColorSecondaryBrush" Color="{ThemeResource SolidBackgroundFillColorSecondary}" />
             <SolidColorBrush x:Key="SolidBackgroundFillColorTertiaryBrush" Color="{ThemeResource SolidBackgroundFillColorTertiary}" />
             <SolidColorBrush x:Key="SolidBackgroundFillColorQuarternaryBrush" Color="{ThemeResource SolidBackgroundFillColorQuarternary}" />
             <!-- Acrylic Resources -->
-            <Color x:Key="SolidBackgroundAcrylic">#332041</Color>
+            <Color x:Key="SolidBackgroundAcrylic">#592A37</Color>
```

Comment:
Any specific reason for the color change here?

Reply:
I just copied the values from an updated theme so that's why they changed.

Reply:
I assume the other resources being removed is due to the same?

Reply:
Correct. This is pretty much a copy of an existing theme.

Reply:
Sorry for the many questions, but is this how users should create new themes? Some resources (such as overriding the selected tab color) are missing now. I'll approve this PR, if you are fine for this being the best way to customize this, feel free to merge this PR.

Reply:
Tabs can still be modified, but standard themes shouldn't need to modify them.